specs/038-neutral-command-vocab/data-model.md

```
@@ -0,0 +1,82 @@
+
+The shell script `spex/scripts/spex-adapt-commands.sh` that performs the transformation.
+
+| Attribute | Type | Description |
+|-----------|------|-------------|
+| inputs | (harness, commands_dir, adapters_dir) | Runtime arguments |
+| flags | --dry-run | Preview mode (FR-010) |
+| output | transformed files or diff | Files in place or stdout diff |
+
+## Relationships
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)